pkg/version/version.go

```diff
@@ -16,7 +16,7 @@ var (
 	// LatestTag is the latest released version plus the dev meta version.
 	// Will be overwritten by the release pipeline
 	// Needs a manual change for every tagged release
-	LatestTag = "1.1.0+dev"
```

Comment:
Why was it `1.1.0` before on the `stable-2.0` branch?

Reply:
Has been overlooked. But that is not an issue, because we do not build daylies.